### PR TITLE
Relocate public files to correct folders

### PR DESCRIPTION
**Pull request by charmian01**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Core, Analytics and Auth have data models which are public but not in `Model` folders in the top-most folder.
Core
MSLog, MSCustomProperties -> Move to `Model` folder.
Analytics
MSEventProperties -> Move to `Model` folder.
Auth
MSUserInformation -> Move to `Model` folder.

## Related PRs or issues

[66071](https://dev.azure.com/msmobilecenter/Mobile-Center/_workitems/edit/66071)

### CODE REVIEW

File: AppCenter/AppCenter.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1475,6 +1475,10 @@
 		3844FF161E8C261D003E9194 /* Model */ = {
 			isa = PBXGroup;
 			children = (
+				3592ABA61DC90E3600EF4592 /* MSLogger.m */,
```
Comment (jaelim-ms) [inline AppCenter/AppCenter.xcodeproj/project.pbxproj]: This shouldn't be here. I meant `MSLog`, not `MSLogger`. You should be able to find `MSLog.h` file that isn't under `Model` folder. Please look at the file in Finder.

File: AppCenter/AppCenter.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1475,6 +1475,10 @@
...
+				358F9BC22019531F00B9E22C /* MSLogger.h */,
+				F803BBF11E8E3677004B1E7A /* MSCustomProperties.h */,
+				F803BBF21E8E3677004B1E7A /* MSCustomProperties.m */,
 				3844FF1B1E8C2716003E9194 /* MSDevice.h */,
 				3844FF1A1E8C2716003E9194 /* MSDevice.m */,
 				6E3E2C9D1D35701000B1EE50 /* MSLog.h */,
```
Comment (charmian01) [inline AppCenter/AppCenter.xcodeproj/project.pbxproj]: @jaelim-ms  I see the `MSLog.h` is under `Model` folder, is this not the correct location?

File: AppCenter/AppCenter.xcodeproj/project.pbxproj (hunk 1 above)
Comment (charmian01) [reply AppCenter/AppCenter.xcodeproj/project.pbxproj]: Reset the `MSLogger` location.

File: AppCenter/AppCenter.xcodeproj/project.pbxproj (hunk 2 above)
Comment (jaelim-ms) [reply AppCenter/AppCenter.xcodeproj/project.pbxproj]: `MSLog` is in `Model` folder when you look at this one in Xcode but physical location of the file isn't in `Model`.

Comment (jaelim-ms) [reply AppCenter/AppCenter.xcodeproj/project.pbxproj]: I pushed a commit for my comment. This is what I meant.
https://github.com/microsoft/appcenter-sdk-apple/pull/1718/commits/4ff5f05445086ae1e4ef40bcae98aeb8695747ad

**Pull request merged**